src/MultiFormGoals/MultiFormGoal/Shortcode.php

```diff
@@ -73,7 +71,7 @@ protected function parseAttributes( $pairs, $attributes ) {
 
 		if ( $attributes ) {
 			foreach ( $attributes as $key => &$attribute ) {
-				if ( is_array( $pairs[ $key ] ) ) {
+				if ( isset( $pairs[ $key ] ) && is_array( $pairs[ $key ] ) ) {
```

Comment:
Can you please explain the changes being made here? This seems like an out-of-scope fix.

Reply:
Yes, that was a separate issue https://github.com/impress-org/givewp/issues/5564 
I don't know how that changes are merged into this PR.